mipengine/common/sql_injection_guard.py

```diff
@@ -0,0 +1,42 @@
+import re
+from functools import wraps
+
+
+def sql_injection_guard(func):
+    wraps(func)
+
+    def wrapper(*args, **kwargs):
+        all_args = list(args) + list(kwargs.values())
+        for arg in all_args:
+            if type(arg) == str:
```

Comment:
Use `isinstance` instead of equality of `type`, see [here](https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance) for discussion.